### ISSUE TITLE
'buildah rmi dir:/tmp/image1' doesn't work

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [projectatomic/buildah#430]
**Description**

When commiting an image to a local directory, the image can not be removed with 'buildah rm dir:/tmp/image1' assuming the directory /tmp/image1 was the directory that was designated in the commit command.

**Steps to reproduce the issue:**
1. export elsewhere=/tmp/image1
2. mkdir -p ${elsewhere}
3. buildah from --name cnt1 busybox
4. buildah commit cnt1 dir:${elsewhere}
5. buildah rmi dir:${elsewhere}

**Describe the results you received:**
```
 buildah rmi dir:${elsewhere}
could not get image "dir:/tmp/image1": error reading image using reference "dir:/tmp/image1": image not known
```

**Describe the results you expected:**

The image would be removed without an error.
**Output of `rpm -q buildah` or `apt list buildah`:**

**Output of `buildah version`:**

```
buildah version
Version:       0.11
Go Version:    go1.9.2
Image Spec:    1.0.0
Runtime Spec:  1.0.0
Git Commit:    967a045
Built:         Tue Jan 30 08:16:04 2018
OS/Arch:       linux/amd64
```

**Output of `cat /etc/*release`:**

```
# cat /etc/*release
Fedora release 27 (Twenty Seven)
NAME=Fedora
VERSION="27 (Server Edition)"
ID=fedora
VERSION_ID=27
PRETTY_NAME="Fedora 27 (Server Edition)"
ANSI_COLOR="0;34"
CPE_NAME="cpe:/o:fedoraproject:fedora:27"
HOME_URL="https://fedoraproject.org/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=27
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=27
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Server Edition"
VARIANT_ID=server
Fedora release 27 (Twenty Seven)
Fedora release 27 (Twenty Seven)
```

**Output of `uname -a`:**

```
# uname -a
Linux localhost.localdomain 4.14.13-300.fc27.x86_64 #1 SMP Thu Jan 11 04:00:01 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

```

**Output of `cat /etc/containers/storage.conf`:**

```
]# cat /etc/containers/storage.conf
# storage.conf is the configuration file for all tools
# that share the containers/storage libraries
# See man 5 containers-storage.conf for more information

# The "container storage" table contains all of the server options.
[storage]

# Default Storage Driver
driver = "overlay"

# Temporary storage location
runroot = "/var/run/containers/storage"

# Primary Read/Write location of container storage
graphroot = "/var/lib/containers/storage"

[storage.options]
# AdditionalImageStores is used to pass paths to additional Read/Only image stores
# Must be comma separated list.
additionalimagestores = [
]

# Size is used to set a maximum size of the container image.  Only supported by
# certain container storage drivers.
size = ""

# OverrideKernelCheck tells the driver to ignore kernel checks based on kernel version
override_kernel_check = "true"

```

**Comment by TomSweeneyRedHat** [comment]
FYI @ripcurld0

**Comment by TomSweeneyRedHat** [comment]
Closing as issues addressed in #464, thanks for the issue report @ripcurld0
